src/interactiveConcatTextDocument.ts

```diff
@@ -3,285 +3,333 @@
 /* eslint-disable class-methods-use-this */
 
 import {
-    NotebookDocument,
-    Position,
-    Range,
-    TextDocument,
-    Uri,
-    workspace,
-    Event,
-    EventEmitter,
-    Location,
-    TextLine,
-} from 'vscode';
-import { NotebookConcatTextDocument } from 'vscode';
-
-import { IVSCodeNotebook } from './common/types';
-import { InteractiveInputScheme } from './common/utils';
-import { IConcatTextDocument, score } from './concatTextDocument';
+  NotebookDocument,
+  Position,
+  Range,
+  TextDocument,
+  Uri,
+  workspace,
+  Event,
+  EventEmitter,
+  Location,
+  TextLine,
+} from "vscode";

```

Comment:
I didn't change all that much, prettier just stepped in and updated it to look like the jupyter repo